src/Content/Text/BBCode.php

```diff
@@ -209,6 +209,183 @@ function ($match) use ($body, $no_link_desc) {
 		);
 	}
 
+	public static function getAttachedData(string $body, array $item = []): array
```

Comment:
Please add a deprecation notice like you did in the PR description.

Reply:
This is a quick fix, since you currently can't post to Twitter. My plan is to get rid of this function until the weekend or even earlier.